components/form-builder/app/responses/DownloadTableReducer.ts

```diff
@@ -33,7 +32,7 @@ export const initialTableItemsState = (
   return {
     checkedItems: new Map(),
     statusItems: new Map(vaultSubmissions.map((submission) => [submission.name, false])),
-    sortedItems: sortVaultSubmission(vaultSubmissions),
+    sortedItems: vaultSubmissions,
```

Comment:
Does it make sense to still call this sortedItems?

Reply:
yeah not really anymore, will update

Reply:
`allItems`